src/main/java/nextstep/ladder/Main.java

```diff
@@ -1,8 +1,55 @@
 package nextstep.ladder;
 
 
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import nextstep.ladder.domain.Ladder;
+import nextstep.ladder.domain.LadderFactory;
+import nextstep.ladder.domain.Player;
+import nextstep.ladder.domain.PlayerName;
+import nextstep.ladder.domain.PlayerNames;
+import nextstep.ladder.domain.Players;
+import nextstep.ladder.domain.RandomLadderBarStatusDecider;
+import nextstep.ladder.domain.Referee;
+import nextstep.ladder.domain.dto.GameResults;
+import nextstep.ladder.presentation.InputView;
+import nextstep.ladder.presentation.ResultView;
+
 public class Main {
+  private static final InputView inputView = new InputView();
+  private static final ResultView resultView = new ResultView();
+  private static final Referee referee = new Referee();
+
   public static void main(String[] args) {
-    new LadderGame().play();
+    PlayerNames playerNames = inputView.getPlayerNames();
+    GameResults inputGameResults = inputView.getGameResults();
+
+    int ladderLength = inputView.getLadderLength();
+    Ladder ladder = LadderFactory.createLadder(ladderLength, playerNames.size(), new RandomLadderBarStatusDecider());
+    GameResults evaluatedGameResults = referee.getResults(ladder, playerNames, inputGameResults);
```

Comment:
```suggestion
    GameResults evaluatedGameResults = ladder.play();
```
위와 같이 Ladder에 메시지를 보내 구현하는 것은 어떨까?